server/src/cmds/config/cameras.rs

```diff
@@ -647,14 +661,47 @@ fn edit_camera_dialog(db: &Arc<db::Database>, siv: &mut Cursive, item: &Option<i
                 |v: &mut views::TextView| v.set_content("<new>"),
             );
         }
-        dialog.title("Add camera").button("Add", {
-            let db = db.clone();
-            move |s| press_edit(s, &db, None)
-        })
+        dialog
+            .title("Add camera")
+            .button("Add", {
+                let db = db.clone();
+                move |s| press_edit(s, &db, None)
+            })
+            .button("Copy config", {
```

Comment:
Can we only show this if there are existing cameras to copy?

Reply:
Only showing if there are existing configurations to copy has the downside in that the first time the user sees the screen and is learning how it works, they wouldn't learn that the option exists. Noticing that the option exists would require them noticing that their options have changed, after they likely assume they understand how it works and might be skimming over the fields. I'm not sure I like the idea of the UX impact of introducing this inconsistency in the interface.

Reply:
Good point; I suppose disabling the button is better than completely hiding it.